server/fleet/statistics.go

```diff
@@ -21,6 +21,7 @@ type StatisticsPayload struct {
 	NumWeeklyActiveUsers           int                                `json:"numWeeklyActiveUsers"`
 	HostsEnrolledByOperatingSystem map[string][]HostsCountByOSVersion `json:"hostsEnrolledByOperatingSystem"`
 	StoredErrors                   json.RawMessage                    `json:"storedErrors"`
+	NumHostsNotResponding          int                                `json:"numHostsNotResponding"`
```

Comment:
We should start documenting these fields, for some of them it's not obvious what they are.

Reply:
Any thoughts on the best place to include this documentation?

Reply:
For starters, right here in each exported member (e.g. it will not be obvious for a new developer what `NumHostsNotResponding` is).